content/docs/self-hosted-runners.md

```diff
@@ -1,82 +1,81 @@
-# Self-hosted Runners
+# Self-hosted (On-premise or Cloud) Runners
 
-GitHub Actions and GitLab CI are run on GitHub- and GitLab- hosted runners by
-default. However, there are many great reasons to use your own runners: to take
-advantage of GPUs; to orchestrate your team's shared computing resources, or to
-train in the cloud.
+GitHub Actions and GitLab CI workflows are run on GitHub- and GitLab- hosted
+runners by default. However, there are many great reasons to use your own
+runners: to take advantage of GPUs; to orchestrate your team's shared computing
+resources, or to train in the cloud.
```

Comment:
```suggestion
runners: to take advantage of GPUs, to orchestrate your team's shared computing
resources, or to train in the cloud.
```

Not quite familiar with the usage of semicolons in English as list delimiters, but [it looks](https://www.bbc.co.uk/bitesize/topics/zr6bxyc/articles/zhntng8) like it shouldn't be necessary on this sentence and, even if it were, it should be used consistently, also on the last element.

About the last element consistency I'm not completely sure, as I have seen it used both ways (i.e. “item; item, and item” and “item; item; and item”) in many texts. 🙃

Reply:
reworded